src/components/ResumePage.astro

```diff
@@ -1,7 +1,13 @@
 ---
 import PageHeader from '@components/PageHeader.astro';
+import { ResumePageContainer } from '@components/ResumePageContainer';
 ---
 
-<PageHeader pageKey="resume" />
+<div class="flex flex-col justify-center items-center w-full">
+  <PageHeader pageKey="resume" />
+  <div class="flex-grow overflow-auto">
+    <ResumePageContainer />
+  </div>
+</div>
```

Comment:
I want to keep building on this- not quite sure what the structure will be yet- some sort of scrollable or tabbable container to display both of their resumes